src/debug/debug.cc

```diff
@@ -3787,17 +3787,19 @@ char* CommandCallback(const char* command, const char* params) {
   }
 
   Handle<Object> result = rv.ToHandleChecked();
+  Handle<Object> rvStr = JsonStringify(isolate, result, undefined, undefined).ToHandleChecked();
+  std::unique_ptr<char[]> rvCStr = String::cast(*rvStr).ToCString();
 
-  if (result->IsException() || result->IsJSError()) {
-    // TODO: Just trying to naively convert the result into a json string doesn't produce useful results.
-    // Figure out how to get the message (and a stack) from it so that we can tell the controller what's
-    // going on.
-    recordreplay::Diagnostic("Error: CommandCallback %s failed with exception", command);
-    IMMEDIATE_CRASH();
+  Handle<String> is_error_string = isolate->factory()->NewStringFromAsciiChecked("is_error");
+  MaybeHandle<Object> is_error_value = Object::GetProperty(isolate, result, is_error_string);
+  if (!is_error_value.is_null()) {
+    if (is_error_value.ToHandleChecked()->BooleanValue(isolate)) {
+      recordreplay::Print("ErrorFatal %s:%d %s", "js", 0, rvCStr.get());
```

Comment:
Can this have a comment linking to a linear issue for fixing this properly, as with the place where @kannanvijayan added a similar print recently?  I'm nervous about having this stuff sprinkled around the code base without any clear references to where we're printing these strings that will be interpreted by the backend in a particular way.

Also, I don't understand what the is_error property is or where it's assigned.  Does this tie into some upcoming chromium changes?  Can the comment here explain what is going on?

Reply:
Sure thing, I'll update the code momentarily.  This is to facilitate a second change which will be in chromium proper, to return an error object instead of the exception, so that we can format it as above to get our stacktrace.

@bhackett1024 In general, how should one go about putting out dual prs in chrome and v8?  Domi suggests just putting them both out at the same time, with one linking to the other, and then updating the chrome pr when the v8 change lands with deps changes.  Does this sound good?

Reply:
@klochek sure, that sounds good